<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Teacher_win2</class>
 <widget class="QDialog" name="Teacher_win2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>740</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1047</width>
    <height>740</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:transparent</string>
  </property>
  <widget class="QPushButton" name="btnEndClass">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>650</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 172, 146);
border-radius:10px;
border-width: 1px;
border-color: rgb(0, 0, 0);
border-style: solid;</string>
   </property>
   <property name="text">
    <string>End Class</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>160</y>
     <width>121</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,1,0,1,0,1,0">
    <property name="spacing">
     <number>7</number>
    </property>
    <item>
     <widget class="QPushButton" name="DiapatchQuesBtn">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>61</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>61</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>在线发题</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RandomCallBtn">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>61</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>61</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>随机语音提问</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ScreenShareBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>61</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>61</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>共享屏幕</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="VoiceShareBtn">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>61</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>61</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>共享语音</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="StuTWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>581</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
border-width: 2px;
border-color: rgb(0, 0, 0);
border-style: solid;
background-color:rgb(239, 239, 239);</string>
   </property>
   <column>
    <property name="text">
     <string>学生列表</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>状态</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="whoIsTalkingLbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>no one else is called.</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>220</y>
     <width>111</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1">
    <property name="spacing">
     <number>1</number>
    </property>
    <item>
     <widget class="QPushButton" name="StopRandomCallBtn">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">backgroung-color:transparent;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="StopScreenBtn">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="StopVoiceBtn">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>StuTWidget</zorder>
  <zorder>whoIsTalkingLbl</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>btnEndClass</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
